src/main/ipc_handler.ts

```
@@ -60,4 +60,8 @@
   ipcMain.handle("writeClipboardText", (_event: IpcMainInvokeEvent, text: string) => {
     clipboard.writeText(text ?? "");
   });
+
+  ipcMain.handle("updateInstall", (_event: IpcMainInvokeEvent) => {
```

ESLint

[error] '_event' is defined but never used. Allowed unused args must match /^__/u.

(@typescript-eslint/no-unused-vars)